pkg/model/image_manager.go

```diff
@@ -11,13 +11,15 @@ const (
 	RHSSOImageOpenJDK     = "RELATED_IMAGE_RHSSO_OPENJDK"
 	RHSSOImage            = "RELATED_IMAGE_RHSSO"
 	KeycloakInitContainer = "RELATED_IMAGE_KEYCLOAK_INIT_CONTAINER"
+	RHSSOInitContainer    = "RELATED_IMAGE_RHSSO_INIT_CONTAINER"
 	RHMIBackupContainer   = "RELATED_IMAGE_RHMI_BACKUP_CONTAINER"
 	PostgresqlImage       = "RELATED_IMAGE_POSTGRESQL"
 
 	DefaultKeycloakImage         = "quay.io/keycloak/keycloak:9.0.2"
-	DefaultRHSSOImageOpenJ9      = "registry.redhat.io/rh-sso-7/sso74-openshift-rhel8:7.4-1"
-	DefaultRHSSOImageOpenJDK     = "registry.redhat.io/rh-sso-7/sso74-openshift-rhel8:7.4-1"
+	DefaultRHSSOImageOpenJ9      = "registry.redhat.io/rh-sso-7/sso74-openj9-openshift-rhel8:7.4"
```

Comment:
Do we want this tag? As the default, we should probably have some specific build instead of the latest. WDYT? The same for OpenJDK image.

Reply:
I would like to preserve pinning (at least to some extends). With the `latest`, failures may come from nowhere....

Reply:
Yeah, that's exactly what I am afraid about with tag `7.4`. If I remember correctly tag `7.4` works as the latest 7.4 build. @hmlnarik could you confirm, please?

Reply:
That's my understanding, and per my understanding this should be here as suggested.